elements/lisk-transaction-pool/test/unit/transaction_list.spec.ts

```diff
@@ -310,7 +330,7 @@ describe('TransactionList class', () => {
 				});
 			});
 
-			describe('when new transaction is added with higher nonce than the existing highest nonce', () => {
+			describe('when new transaction is added with higher nonce than the existing highest nonce when transactionList.size >= maxSize', () => {
```

Comment:
As Shu pointed out before, we are already inside the case, so `when transactionList.size >= maxSize` is not necessary here